mozetl/search/daily_search_rollups.py

```diff
@@ -0,0 +1,296 @@
+"""
+# Daily main_summary to Vertica Search Rollups
+
+This job is added by Bug 1364530 and was originally located
+[1]. Search rollups are computed and then ingested by Vertica.
+
```

Comment:
Please note that this job has not been reviewed or tested. Previous discussion here [2]. We have an open bugs for tests here [3].


[2] https://github.com/mozilla/mozilla-reports/pull/23
[3] https://bugzilla.mozilla.org/show_bug.cgi?id=1365653